source/jormungandr/jormungandr/interfaces/v1/GraphicalIsochrone.py

```diff
@@ -63,78 +64,27 @@
         parser_get = self.parsers["get"]
-
-        parser_get.add_argument("from", type=unicode, dest="origin")
-        parser_get.add_argument("first_section_mode[]",
-                                type=option_value(modes),
-                                dest="origin_mode", action="append")
-        parser_get.add_argument("last_section_mode[]",
-                                type=option_value(modes),
-                                dest="destination_mode", action="append")
-        parser_get.add_argument("to", type=unicode, dest="destination")
-        parser_get.add_argument("datetime", type=date_time_format)
-        parser_get.add_argument("max_duration", type=unsigned_integer)
         parser_get.add_argument("min_duration", type=unsigned_integer, default=0)
-        parser_get.add_argument("forbidden_uris[]", type=unicode, action="append")
-        parser_get.add_argument("max_transfers", type=int, default=42)
-        parser_get.add_argument("_current_datetime", type=date_time_format, default=datetime.utcnow(),
-                                description="The datetime used to consider the state of the pt object"
-                                            " Default is the current date and it is used for debug."
-                                            " Note: it will mainly change the disruptions that concern "
-                                            "the object The timezone should be specified in the format,"
-                                            " else we consider it as UTC")
-        parser_get.add_argument("max_walking_duration_to_pt", type=int,
-                                description="maximal duration of walking on public transport in second")
-        parser_get.add_argument("max_bike_duration_to_pt", type=int,
-                                description="maximal duration of bike on public transport in second")
-        parser_get.add_argument("max_bss_duration_to_pt", type=int,
-                                description="maximal duration of bss on public transport in second")
-        parser_get.add_argument("max_car_duration_to_pt", type=int,
-                                description="maximal duration of car on public transport in second")
-        parser_get.add_argument("walking_speed", type=float_gt_0)
-        parser_get.add_argument("bike_speed", type=float_gt_0)
-        parser_get.add_argument("bss_speed", type=float_gt_0)
-        parser_get.add_argument("car_speed", type=float_gt_0)
 
     @marshal_with(graphical_isochrones)
     @ManageError()
-    def get(self, region=None):
-        args = self.parsers['get'].parse_args()
+    def get(self, region=None, uri=None):
 
-        if args.get('origin_mode') is None:
-            args['origin_mode'] = ['walking']
-        if args.get('destination_mode') is None:
-            args['destination_mode'] = ['walking']
+        args = self.parsers['get'].parse_args()
+        resp = JourneyCommon.get(self, region, uri)
```

Comment:
If you change the name of the method, you can write this `resp = self.parse_args(region, uri)`